standard/expressions.md

```diff
@@ -1459,6 +1492,45 @@ The run-time processing of a delegate invocation of the form `D(A)`, where `D`
 
 See [§20.6](delegates.md#206-delegate-invocation) for details of multiple invocation lists without parameters.
 
+### §null-conditional-invocation-expression Null Conditional Invocation Expression
+
+A *null_conditional_invocation_expression* consists of a *primary_expression* followed by a "`?(`" token, followed by an optional *argument_list*, followed by a "`)`" token.
```

Comment:
The `?` and `(` are separate tokens.

Reply:
But I think it isn't implemented because of an ambiguity with the `?:` expression.

Reply:
I'll "resolve conversation" this one as it's dealt with by the preceding thread (both operators should be handled the same way!)